recipes/trajan/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "trajan" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7b44ccd9898f45ac04033e46158f01748962ba9a943329ba0c0e27b59743296
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - trajanshow = trajan.scripts.trajanshow:main
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - poetry
+  run:
```

Comment:
Many of these dependencies have minimum versions in the upstream `setup.py`/`pyproject.toml`. We generally would like to see matching restrictions here.

Reply:
Done.